centrallix-lib/lcov.info

```diff
@@ -0,0 +1,7337 @@
+TN:
```

Comment:
Can you remove this file from the branch? It's not relevant to the website changes